README.md

```diff
@@ -83,6 +90,34 @@ The following options can be used to further configure the processing:
 If no option is set, the tool performs layout detection of main regions (background, text, images, separators and marginals).
 The best output quality is produced when RGB images are used as input rather than greyscale or binarized images.
 
+### Binarization
+Document Image Binarization
+
+The command-line interface for binarization of single image can be called like this:
+
+```sh
+eynollah binarization \
+  -m <path to directory containing model files> \
+  <input image> \
+  <output image>
```

Comment:
Does `<output image>` here expect a filename or a path? Because when it uses the path it would be nicer to be consistent with the CLI documentation for layout analysis and also call this `<output directory>`.

Reply:
Binarization for a single image requires a filename (preferably in PNG format). As you suggested, I can update this to accept a file path instead. However, this change would also affect the OCR-D framework. What are your thoughts on this, @kba?

Reply:
We can adopt the OCR-D processor if the API changes but IIUC this would only be about the CLI, i.e. replacing the arguments with the same `-i/-o` options as `eynollah layout`? If so, then fine by me.